test-end-to-end/android-maestro/common/edit_recorded_attendance.yaml

```diff
@@ -0,0 +1,19 @@
+appId: com.toughra.ustadmobile
+---
+- tapOn: "Courses"
+- tapOn:
+      id: "com.toughra.ustadmobile:id/item_clazzlist_clazz_cl"
+- tapOn: "ATTENDANCE"
+- tapOn:
+    point: "58%,81%"
```

Comment:
Why this exact point? Are we sure these coordinates will always work?

Reply:
Hi Mike, Yes, using these literal values is not the right practice. ID is available for this element. I have changed the point command to id. Thank you.